client/src/components/add-job-lead-component/jobLeadCard.jsx

```diff
@@ -18,7 +18,7 @@ import dayjs from "dayjs";
 import DeleteIcon from "@mui/icons-material/Delete";
 import * as React from "react";
 import { JobLeadContainer, H3 } from "./index.styles";
-import { JOB_TYPES } from "../../utils/contants";
+import { JOB_TYPES } from "../../utils/constants";
```

Comment:
I see "Base & Commission" being used. I'm not sure if this would be a problem, but the backend only accepts "Base and Commission" due to ampersands not being allowed to be used in ENUMS in MySQL.